lib/index.ts

```diff
@@ -3,10 +3,12 @@ const mod = require('module'); // Reasons:tm:
 import * as path from 'path';
 import * as vm from 'vm';
 
+const cache = Object.create(null);
```

Comment:
I think that having a shared cache will break the encapsulation?

Perhaps it needs to be instantiated per call, and passed between?